lighthouse-core/gather/drivers/driver.js

```diff
@@ -492,7 +492,8 @@ class Driver {
   beginEmulation() {
     return Promise.all([
       emulation.enableNexus5X(this),
-      emulation.enableNetworkThrottling(this)
+      emulation.enableNetworkThrottling(this),
```

Comment:
while you're in there... :)

can you add the `--disable-device-emulation` flag as well? So you'll be adding all three: `--disable-device-emulation --disable-cpu-throttling --disable-network-throttling`